File: Arrays/helper_files/Array.cpp
#include <iostream>
using namespace std;

class Array {

    protected:
        int n;
        int *arr = new int[n];
        
    public:
        void getArrayFromUser(){
            cout << "Enter the size of the array: ";
            cin >> n;
            for (int i = 0; i < n; i++) {
                cin >> arr[i];
            }
        }

    public:
        void printArray() {
            for (int i = 0; i < n; i++) {
                cout << arr[i] << " " ;
            }
        }

      public:
        int getSizeOfArray(){
            return n;
        }

};

File: Arrays/helper_files/Array.java
package Arrays.helper_files;

import java.util.Arrays;

import Java_helper_classes.FastReader;

public class Array {
    
    int n; 
    protected int[] arr; //declaring array

    public void getArrayFromUser(){
        FastReader s = new FastReader();
        System.out.println("Enter the size of array: ");
        n = s.nextInt();
        arr = new int[n];  // allocating memory to array
        System.out.println("Enter the elements of array: ");
        for (int i = 0; i < n; i++) {
            arr[i] = s.nextInt(); 
        }
    }

    public void printArray(){
        
        System.out.println(Arrays.toString(arr));

    }



}
